src/tests/Legend.spec.js

```diff
@@ -62,12 +62,14 @@ describe('The Legend component', function () {
         },
       },
     });
-    var lis = TestUtils.scryRenderedDOMComponentsWithTag(this.renderedComponent, 'li');
-    expect(lis.length).toBe(3);
-    var spans = TestUtils.scryRenderedDOMComponentsWithTag(lis[1], 'span');
+
+    var lis = ReactDOM.findDOMNode(this.renderedComponent);
```

Comment:
:question: Won't `this.renderedComponent` already be a DOM Node at this point; can we delete the `findDOMNode` call?